test/Common/TestHelpers.cs

```diff
@@ -111,6 +113,98 @@ public static string GetTaskHubNameFromTestName(string testName, bool enableExte
             return truncatedTestName + deterministicSuffix + randomSuffix;
         }
 
+        /// <summary>
+        /// Returns <c>true</c> if a JSON Durable log has all the minimum-required keys.
+        /// Else, returns <c>false</c>.
+        /// </summary>
+        /// <param name="json">The JSON log to validate.</param>
+        public static bool IsValidJSONLog(JObject json)
+        {
+            List<string> expectedKeys = new List<string>
+            {
+                "EventStampName",
+                "EventPrimaryStampName",
+                "ProviderName",
+                "TaskName",
+                "EventId",
+                "EventTimestamp",
```

Comment:
Probably out of scope for this PR, but with this change (or with changes that preceded it) will we get fractional seconds just like we do with Windows? Not being able to properly sort by timestamps has been killing me when doing debugging on Linux. :)

Reply:
Ah yes, that's killing me as well. Unfortunately, the answer is no, and I explain more about why in here: https://github.com/Azure/azure-functions-durable-extension/issues/1843

Let's use that issue to track and discuss this work-item :)